CMakeLists.txt

```
@@ -125,12 +125,12 @@ set(TEST_INCLUDE_DIRECTORIES
 ## Input files
 ###############################################################################
 
-file(GLOB_RECURSE C_SOURCES CONFIGURE_DEPENDS "src/*.c")
-file(GLOB_RECURSE C_HEADERS CONFIGURE_DEPENDS "inc/*.h")
+file(GLOB_RECURSE C_SOURCES CONFIGURE_DEPENDS "src/*.cpp")
+file(GLOB_RECURSE C_HEADERS CONFIGURE_DEPENDS "inc/*.hpp")
```

```suggestion
file(GLOB_RECURSE C_SOURCES CONFIGURE_DEPENDS "src/*.cpp" "src/*.c")
file(GLOB_RECURSE C_HEADERS CONFIGURE_DEPENDS "inc/*.hpp" "inc/*.h)
```
Acredito eu que o template, por ser bem genericão, ainda deva suportar C. Pode ser que precisemos por algum motivo de alguns arquivos em C, ou pode ser que num futuro seja mais adequado se utilizar de C para algum projeto. No fim do dia, não  faz mal manter o suporte para C.